src/utilities/delta_util.ts

```diff
@@ -184,44 +170,44 @@ export class ControlDelta {
     result.clean();
     return result;
   }
+}
 
-  /**
-   * Get the changes in the controls individual segments.
-   * They are returned as a list of change groups, with each group encoding a segment.
-   */
-  get segment_changes(): ControlChangeGroup[] {
-    // Change in individual control segments
-    let old_segs = this.old.root.hdf.segments;
-    let new_segs = this.new.root.hdf.segments;
-    if (old_segs === undefined || new_segs === undefined) {
-      // Oh well
-      return [];
-    }
-
-    // Pair them by position. Crude but hopefully fine
-    // Abort if they aren't the same length
-    if (old_segs.length !== new_segs.length) {
-      console.warn('Unable to match control segments for delta');
-      return [];
-    }
-
-    // Do the actual pairing/diff fingind
-    let results: ControlChangeGroup[] = [];
-    for (let i = 0; i < old_segs.length; i++) {
-      let old_seg = old_segs[i];
-      let new_seg = new_segs[i];
-      let changes = changelog_segments(old_seg, new_seg);
-      let group = new ControlChangeGroup(old_seg.code_desc, changes);
-
-      // Clean it up and store if not empty
-      group.clean();
-      if (group.any) {
-        results.push(group);
+export function get_eval_start_time(
+  ev: ContextualizedEvaluation
+): string | null {
+  for (let prof of ev.contains) {
+    for (let ctrl of prof.contains) {
+      if (ctrl.hdf.segments!.length) {
+        let t = ctrl.hdf.segments![0].start_time;
+        return t;
```

Comment:
No need for this to be 2 lines, just return the time. `return ctrl.hdf.segments![0].start_time;`